libraries/lib-ffmpeg-support/wrappers/AVIOContextWrapper.cpp

```diff
@@ -35,6 +35,7 @@ AVIOContextWrapper::operator=(AVIOContextWrapper&& rhs) noexcept
 {
    std::swap(mFFmpeg, rhs.mFFmpeg);
    std::swap(mAVIOContext, rhs.mAVIOContext);
+   std::swap(mpFile, rhs.mpFile);
```

Comment:
I think I would just explicitly delete the move constructor/assignment here, as well as in most of the other places

Reply:
It generally makes no sense as this is an abstract class and is leftover from the moment, when I was "generating" wrappers